src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -4260,14 +4256,15 @@ private function step_in_foreign_content(): bool {
 		 * > An end tag whose name is "script", if the current node is an SVG script element.
 		 */
 		if ( $this->is_tag_closer() && 'SCRIPT' === $this->state->current_token->node_name && 'svg' === $this->state->current_token->namespace ) {
+			in_foreign_content_svg_script_close_tag:
 			$this->state->stack_of_open_elements->pop();
+			return true;
```

Comment:
This fixes a fall-through to the next conditional block that I believe was unintentional. Typically, we expect to enter a single block.